src/components/Banner/Banner.jsx

```diff
@@ -94,6 +112,8 @@ const Banner = (props) => {
     setCookie('confirm_facebook', 1, options);
     setCookie('confirm_youtube', 1, options);
     setCookie('confirm_google', 1, options);
+    setCookie('confirm_twitter', 1, options);
+    setCookie('confirm_instagram', 1, options);
     setCookie('confirm_cookies', 1, options);
```

Comment:
This is kind of a separate issue, but the "confirm all" feature should only set cookies for the modules that are enabled. Otherwise if a site adds a module that was previously disabled, a user might find that they have the cookie already without ever opting in!

Reply:
@danalvrz sorry, I didn't explain very well. We should keep all the modules here, but we should add a check for each one e.g
```
if (includes(modules, 'twitter')) setCookie('confirm_twitter', 1, options);
```

or more concise, do it for all the modules in a loop:

```
modules.forEach((module) => setCookie(`confirm_${module}`, 1, options));
setCookie('confirm_cookies', 1, options);
```